cluster-up/cluster/kind-1.25-sriov/certcreator/certsecret.go

```diff
@@ -0,0 +1,159 @@
+package main
```

Comment:
i think you can delete this folder we dont use cert creator anymore

Reply:
Lets get to improvements after we'll have the provider working :)

Reply:
you can np, but this folder is not used so it should be an easy one